src/routes/sign/Sign.svelte

```diff
@@ -1,23 +1,26 @@
 <script lang="ts">
     import Bridge from '../../lib/bridge/Bridge';
     import Transaction from '../../lib/components/transaction/Transaction.svelte';
-    import { language } from '../../store/global';
-
+    import { language, transaction } from '../../store/global';
```

Comment:
transaction should be part of the signStore, rather than the global store, is my understanding